src/TimeService.Broker/Requests/DepartmentService.cs

```diff
@@ -119,5 +123,19 @@ public async Task<List<DepartmentFilteredData>> GetDepartmentFilteredDataAsync(L
 
       return departmentsData;
     }
+
+    public async Task<DepartmentUserRole?> GetDepartmentUserRoleAsync(
+      Guid departmentId,
+      Guid userId,
+      List<string> errors = null)
```

Comment:
Isn't it null by default?

Reply:
in this case it is null by default because this method is called via interface, where it is null too
in other cases it is not